.pre-commit-config.yaml

```diff
@@ -1,29 +1,34 @@
 repos:
   - repo: "https://github.com/PyCQA/flake8"
-    rev: "7.0.0"
+    rev: "7.1.1"
     hooks:
       - id: "flake8"
         name: "Python: analysis"
 
   - repo: "https://github.com/pre-commit/mirrors-mypy"
-    rev: "v1.8.0"
+    rev: "v1.13.0"
     hooks:
       - id: "mypy"
         name: "Python: types"
 
-  - repo: https://github.com/pycqa/isort
+  - repo: "https://github.com/PyCQA/isort"
     rev: "5.13.2"
     hooks:
-      - id: isort
-        args: [ "--profile", "black"]
+      - id: "isort"
```

Comment:
Might become a problem in the future since `pre-commit clean && pre-commit run --all` gives a warning:

```
[INFO] Initializing environment for https://github.com/PyCQA/isort.
[WARNING] repo `https://github.com/PyCQA/isort` uses deprecated stage names (commit, merge-commit, push) which will be removed in a future version.  Hint: often `pre-commit autoupdate --repo https://github.com/PyCQA/isort` will fix this.  if it does not -- consider reporting an issue to that repo.
```

Unfortunately the project haven’t seen updates for almost a year. There is [an alternative](https://github.com/asottile/reorder-python-imports) though.

Reply:
thanks for bringing this up. Would you like to submit follow-up PR (or separate commit here) to migrate from isort to that?

Reply:
Sure thing, I’ll check it out.